bcda/main.go

```diff
@@ -56,6 +57,7 @@ func init() {
 		log.Info("Failed to open error log file; using default stderr")
 	}
 	monitoring.GetMonitor()
+	log.Info(fmt.Sprintf(`Auth is made possible by %T`, reflect.TypeOf(auth.GetProvider())))
```

Comment:
In the logs when I run this locally, I get: `{"level":"info","msg":"Auth is made possible by *reflect.rtype","time":"2019-02-12T13:46:21Z"}`